Xamarin.Forms.Platform.Android/Renderers/FormsTextView.cs

```diff
@@ -8,34 +8,28 @@ namespace Xamarin.Forms.Platform.Android
 {
 	public class FormsTextView : TextView
 	{
-		bool _skip;
-
 		public FormsTextView(Context context) : base(context)
 		{
 		}
 
+		[Obsolete]
 		public FormsTextView(Context context, IAttributeSet attrs) : base(context, attrs)
 		{
 		}
 
+		[Obsolete]
 		public FormsTextView(Context context, IAttributeSet attrs, int defStyle) : base(context, attrs, defStyle)
 		{
 		}
 
+		[Obsolete]
 		protected FormsTextView(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
 		{
 		}
 
-		public override void Invalidate()
```

Comment:
Having this override is what is causing this exception
```
12-28 02:05:39.313 I/MonoDroid(14636): System.NotSupportedException: Unable to activate instance of type Xamarin.Forms.Platform.Android.FastRenderers.LabelRenderer from native handle 0xbfb6fd7c (key_handle 0xd85688d). ---> System.MissingMethodException: No constructor found for Xamarin.Forms.Platform.Android.FastRenderers.LabelRenderer::.ctor(System.IntPtr, Android.Runtime.JniHandleOwnership) ---> Java.Interop.JavaLocationException: Exception of type 'Java.Interop.JavaLocationException' was thrown.
```

The pure existence of the override is causing Android to call it after the View has already been disposed. If you just don't have the override then it never gets registered with XA and XA never tries to call it after dispose. I'm not sure why the invalidate is called in such a delayed fashion though, I'm still validating that with the XA team